Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:slim-buster AS builder
+
+WORKDIR /opt/
+COPY . .
+RUN apt-get update && apt-get install -y binutils libc-bin
+RUN mkdir -p /var/log/ot && pip3 install --no-cache --upgrade -r requirements.txt
+RUN pyinstaller --paths=lib scripts/schedule_resources.py --onefile
+
+FROM gcr.io/distroless/python3-debian11 AS deployer
```

Comment:
We are referring to latest, we should always go with versioned image

Reply:
distroless dont provide tags other than latest